function-controller/pkg/controller/deployer.go

```diff
@@ -93,7 +93,11 @@ func (d *deployer) buildMainContainer(function *v1.Function) corev1.Container {
 
 func (d *deployer) buildSidecarContainer(function *v1.Function) corev1.Container {
 	c := corev1.Container{Name: "sidecar"}
-	c.Image = sidecarImage + ":" + os.Getenv("RIFF_FUNCTION_SIDECAR_TAG")
+	imageName := os.Getenv("RIFF_FUNCTION_SIDECAR_IMAGE")
```

Comment:
I would call this  RIFF_FUNCTION_SIDECAR_REPOSITORY to be consistent with Helm usage

Reply:
ok - done - consistency is good (although i always found the use of the "repository" term on dockerhub confusing)